pkg/sliceutils/diff.go

```diff
@@ -0,0 +1,43 @@
+package sliceutils
+
+// Diff returns, given two slices a and b sorted according to lessFunc, a slice of the elements occurring in a and b
+// only, respectively.
+func Diff[T any](a, b []T, lessFunc func(a, b T) bool) (aOnly, bOnly []T) {
+	i, j := 0, 0
```

Comment:
similarly, here. I feel like `var i, j int` is usually preferred in our codebase

Reply:
done